webapp/src/main/webapp/images/.gitignore

```diff
@@ -1,45 +1,12 @@
-/add.gif
-/alert.gif
-/alertPlus.gif
-/collapse_icon.gif
-/copy.gif
-/delete.gif
-/edit.gif
-/error.gif
-/expand_icon.gif
-/female.gif
-/file.gif
-/gradient.gif
-/help.gif
-/info.gif
-/leftArrow.gif
-/lock.gif
-/lookup.gif
-/male.gif
-/markRead.gif
-/movedown.gif
-/moveup.gif
-/note.gif
-/open.gif
-/openmrs_green_bar_repeat.gif
-/openmrs_green_bar.gif
-/openmrs_logo_short.gif
-/openmrs_logo_text_small.gif
-/openmrs_logo_text_small.png
-/openmrs_logo_tiny.png
-/openmrs_logo_white_large.png
-/openmrs_orange_bar_repeat.gif
-/openmrs_orange_bar.gif
-/openmrs_purple_bar_repeat.gif
-/openmrs_purple_bar.gif
-/patient_F.gif
-/patient_M.gif
-/pdf.gif
-/play.gif
-/printer.gif
-/problem.gif
-/ribbon.jpg
-/rightArrow.gif
-/save.gif
-/stop.gif
-/trash.gif
+*.gif
```

Comment:
I think you need to ignore everything regardless of the extension except for the images in this folder in core